src/reference/90-Developers-Guide/09-Launcher/01-Launcher-Getting-Started.md

```diff
@@ -154,7 +163,9 @@ class Main extends xsbti.AppMain
 Next, define a configuration file for the launcher. For the above class,
 it might look like:
 
-Then, `publishLocal` or `+publishLocal` the application to make it
+FIXME
```

Comment:
I could probably work out the needed example myself from the context, but it might be easier for someone more familiar with using launcher applications (@eed3si9n or @n8han? :smile:).